<ui version="4.0" >
 <class>RouteDlg</class>
 <widget class="QDialog" name="RouteDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Route</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <widget class="QPushButton" name="remove_last_item_btn" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>Remove last item</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:8pt;">To add items to the&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:8pt;">route, simply click&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:8pt;">them with  the left &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:8pt;">mouse button inside&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:8pt;">the main chart window.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item rowspan="4" row="0" column="1" >
    <widget class="QTableWidget" name="routelist" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text" >
       <string>Pos</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Labels</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>110</width>
       <height>60</height>
      </size>
     </property>
     <property name="title" >
      <string>Routename</string>
     </property>
     <widget class="QLineEdit" name="routename" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="routetype_box" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>110</width>
       <height>135</height>
      </size>
     </property>
     <property name="title" >
      <string>Routetype</string>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>30</y>
        <width>70</width>
        <height>96</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QRadioButton" name="routetype_star" >
         <property name="text" >
          <string>STAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="routetype_sid" >
         <property name="text" >
          <string>SID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="routetype_airway" >
         <property name="text" >
          <string>Airway</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="routetype_unknown" >
         <property name="text" >
          <string>Unknown</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>routename</tabstop>
  <tabstop>routetype_star</tabstop>
  <tabstop>routetype_sid</tabstop>
  <tabstop>routetype_airway</tabstop>
  <tabstop>routetype_unknown</tabstop>
  <tabstop>routelist</tabstop>
  <tabstop>remove_last_item_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
